ci/jenkins/templates/conformance-template.yaml

```diff
@@ -5,6 +5,7 @@
     days-to-keep: 30
     branch: master
     schedule: '3-5'
+    sonobuoy_version: 'v0.18.3'
```

Comment:
without https://build.suse.de/request/show/220338 being accepted this needs to read 0.18.2

Reply:
Good catch. Fixed.